From 1e21459c49c1769d62964f087accca6e9611f1e3 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 12 May 2022 10:43:58 +0200
Subject: [PATCH 186/954] feat(stm32mp2-fdts): add package unique device id in
 STM32MP251

Add the Unique device ID register (96 bits) (UID) in the OTP
definition: BSEC_OTP_DATA5 to 7.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 908da55e3fe5a7fbc05d49e5411c3c153c6daa51 feat(stm32mp2-fdts): add package unique device id in STM32MP251]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I979f8d191e430061b9fa908d4a2cc5663d805ef7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/255793
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp251.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 43a92393e8..5211259375 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -285,6 +285,9 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 
+			uid_otp@14 {
+				reg = <0x14 0xc>;
+			};
 			part_number_otp: otp9@24 {
 				reg = <0x24 0x4>;
 			};
-- 
2.53.0

